orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/pipeline/servergroup/support/TargetServerGroupResolver.groovy

```diff
@@ -159,38 +163,23 @@ class TargetServerGroupResolver {
   }
 
   private <T> T fetchWithRetries(Class<T> responseType, int maxRetries, long retryBackoff, Closure<Response> fetchClosure) {
-    def converter = new JacksonConverter(mapper)
+    return retrySupport.retry({
```

Comment:
Plugging in `retrySupport.retry()` had the side effect of retrying on all exceptions, not just NETWORK and 429.

This change is not particularly necessary but figured it might be worth consolidating our retry behavior somewhat. I couldn't think of a reason why we needed to be selective on what to retry? 

@cfieber Thoughts? I believe you originally authored this.

Reply:
seems fine to me for something that is just doing read requests